### PR TITLE
Prepare for release 1.33.0

### PR DESCRIPTION
**Pull request by amanjeetsingh150**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 1.33.0 - 20223-09-21
+
+- Feature: Adds MAESTRO_DRIVER_STARTUP_TIMEOUT to iOS driver to configure timeout to start iOS driver, used in CI/CD environment with performance limitations. Thanks, @Jesse Farsong for contributing.
```
Comment (axelniklasson) [inline CHANGELOG.md:5]: Is this the right GitHub handle? If not, let's just use the name

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+- Change: Improved Studio's user interface:
+   - Updated fonts to align with company branding.
+   - Introduced a distinct loading animation for better clarity when AI is processing commands.
+- Fix: Crash resulting in Error: No matches found for first query match sequence: `Children matching type Other` due to resolving root element for a snapshot operation on iOS
+- Fix: Android driver getting stuck when the device was disconnected
+- Fix: XCTestUnreachable exceptions due to missing IPv6 config on /etc/hosts
+- Fix: Handling app crash errors from XCUITest drivers gracefully  
+- Fix: Timeouts can be separated with `_`. For example 10_000 for 10000
+
 ## 1.32.0 - 20223-09-06
```
Comment (axelniklasson) [inline CHANGELOG.md:16]: Nit and not related to your changes but let's fix it while we're at it :)

```suggestion
## 1.32.0 - 2023-09-06
```

**Review by axelniklasson — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (amanjeetsingh150) [reply CHANGELOG.md:5]: Nope its name, @ was typo there

File: CHANGELOG.md (hunk 2 above)
Comment (amanjeetsingh150) [reply CHANGELOG.md]: Fixed

**Pull request merged**